examples/how_to/custom/js/children.py

```diff
@@ -2,12 +2,12 @@
 
 import panel as pn
 
-from panel.custom import JSComponent
+from panel.custom import JSComponent, Views
 
 
 class Example(JSComponent):
 
-    children = param.List(item_type=pn.viewable.Viewable)
+    children = Views()
 
     _esm = """
```

Comment:
Curious if this still the best name? Could it be `_code` or something similar?

Reply:
_esm is the name used by AnyWidget. This could be an argument in favor